indexer/editable_map_object.cpp

```diff
@@ -312,6 +312,15 @@ bool EditableMapObject::ValidateEmail(string const & email)
   if (email.empty())
     return true;
 
+  if (strings::IsASCIIString(email))
+    return regex_match(email, regex("^[^@\\s]+@[\\w-]+(\\.[\\w-]+)+$"));
```

Comment:
this regex don't cover several cases for example, the following symbols in name are correct !#$%&'*+-/=?^`_{}|~